src/test/resources/features/allergies.feature

```diff
@@ -0,0 +1,36 @@
+Feature: allergies feature
+
+  Background: 
+    Given User logs in, searches John and visits first patient dashboard
+    And User clicks on edit icon near allergies label
+    Then system loads patients allergies page
+
+  Scenario: Add new allergies
+    And user clicks add new allergy button
+    Then system loads add new allergy page
+    Given User selects one of the three allergen Types options
+    Then corresponding allergens should appear
+    And User selects an allergen
+    And user selects the reactions , enters severity and a comment
+    And clicks Save button
+    Then System should show the allergies of the patient in a table
+
+  Scenario: No known allergies
+    And user clicks on No known allergy button
+    Then No known  allergies should be registered in the table
+
+  Scenario: List allergies
+    Then A list of added allergies should apear in a table
+
+  Scenario: Edit allergies
+    And user clicks on edit icon in the Actions colum of an entry
+    Then system loads Edit allergy page
+    And user makes any changes
```

Comment:
specify at-least one

Reply:
Thank you so much  @k-joseph  for the advice let me work on them and  push again